### PR TITLE
Update 2021 11 30

### PR DESCRIPTION
**Pull request by kba**
Updates core, includes #269 and #261

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,397 @@
...
+  > * Merge branch 'fix-workspace-remove-group-fnfe'
+  > * Merge branch 'rename-group-id'
+  > * remove undeclared requirement on six, fix #726
+  > * bump spec to 3.14.0
+  > * :package: v2.26.1
+  > * Fixed tesseract resource URL for resource 'configs'
+
+### [tesseract](https://github.com/tesseract-ocr/tesseract) [3313bb79](https://github.com/tesseract-ocr/tesseract/commits/3313bb79)..[c676d5bc (rewind)](https://github.com/tesseract-ocr/tesseract/commits/c676d5bc (rewind))
+
+> Release: [5.0.0-alpha-20201231](https://github.com/tesseract-ocr/tesseract/releases/5.0.0-alpha-20201231)
```
Comment (stweil) [inline CHANGELOG.md:28]: There does this release `5.0.0-alpha-20201231` come from? It looks rather old (even older than in the previous `CHANGELOG` entries).

Comment (stweil) [reply CHANGELOG.md:28]: The current Tesseract release is `5.0.0-rc3`. Note that I plan to release `5.0.0` today.

**Review by bertsky — COMMENTED** [review]
Timing-wise (note the last release was over 6 weeks ago) I think we should also include updates to:
- https://github.com/OCR-D/core/pull/747 (after merging)
- https://github.com/OCR-D/core/pull/745 (after merging)
- https://github.com/OCR-D/core/pull/743 (after merging)
- https://github.com/OCR-D/core/pull/739 (after merging)
- https://github.com/OCR-D/ocrd_segment/commit/bdc67719cf4d5e758162ae54c0998f54b9610514
- https://github.com/bertsky/workflow-configuration/commit/af76e39a2c86d67960afc6a3eaf17fbb09cb3b6b

File: CHANGELOG.md (hunk 1 above)
Comment (kba) [reply CHANGELOG.md:28]: > There does this release 5.0.0-alpha-20201231 come from? It looks rather old (even older than in the previous CHANGELOG entries).

Weird indeed. Probably something wrong with the release.sh and my local state. I'll recreate the changelog once PR is done.

**Comment by kba** [comment]
> Timing-wise (note the last release was over 6 weeks ago) I think we should also include updates to:
> 
> * [OCR-D/core#747](https://github.com/OCR-D/core/pull/747) (after merging)
> * [OCR-D/core#745](https://github.com/OCR-D/core/pull/745) (after merging)
> * [OCR-D/core#743](https://github.com/OCR-D/core/pull/743) (after merging)
> * [OCR-D/core#739](https://github.com/OCR-D/core/pull/739) (after merging)
> * [OCR-D/ocrd_segment@bdc6771](https://github.com/OCR-D/ocrd_segment/commit/bdc67719cf4d5e758162ae54c0998f54b9610514)
> * [bertsky/workflow-configuration@af76e39](https://github.com/bertsky/workflow-configuration/commit/af76e39a2c86d67960afc6a3eaf17fbb09cb3b6b)

Preparing a new core release now and will update workflow_configuration and ocrd_segment as suggested, thanks 👍

File: CHANGELOG.md (hunk 1 above)
Comment (stweil) [reply CHANGELOG.md:28]: Tesseract 5.0.0 is now available and can be used here, too.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,69 @@
...
+
+> Release: [5.0.0](https://github.com/tesseract-ocr/tesseract/releases/5.0.0)
+
+  > * Create new release 5.0.0
+  > * Merge pull request #3664 from stweil/classify
+  > * Merge pull request #3663 from stweil/clang7
+  > * autobuild: Fix autogen.sh (reduce build time)
+  > * Remove unused GenericVector::contains_index, UnicityTable::contains_id
+  > * Remove member function GenericVector<T>::contains
+  > * Remove member function UnicityTable<T>::contains
```
Comment (kba) [inline CHANGELOG.md:50]: @stweil, this is still incomplete but at least the link to the released version is now correct. If it bothers you, I can add the complete commit log manually later.

**Review by stweil — APPROVED** [review]
Thanks!

**Comment by bertsky** [comment]
CI is a nightmare these days:
```
Upgrade your pricing plan to take advantage of longer build times

context deadline exceeded
```

I'd say we should parallelize the docker builds, just as we do with the docker pushs.

**Comment by bertsky** [comment]
At last!

**Pull request merged**